packages/builders/__tests__/messages/embed.test.ts

```diff
@@ -134,6 +132,12 @@ describe('Embed', () => {
 			// @ts-expect-error
 			expect(() => embed.setColor('RED')).toThrowError();
 		});
+
+		test('GIVEN an embed with a valid color THEN does not throw error', () => {
```

Comment:
It should test for throwing errors, we already test for not throwing in lines 117-120.